Tests/DependencyInjection/FixtureMonologExtensionTest.php

```diff
@@ -39,9 +39,13 @@ public function testLoadWithSeveralHandlers()
         $this->assertDICDefinitionClass($handler, 'Monolog\Handler\StreamHandler');
         $this->assertDICConstructorArguments($handler, ['/tmp/symfony.log', \Monolog\Logger::ERROR, false, 0666, false]);
 
+        $strategy = $container->getDefinition('monolog.handler.main.error_level_strategy');
```

Comment:
in PHP 7.1 and below your test suite is failing here with `You have requested a non-existent service "monolog.handler.main.error_level_strategy".`. You might want to add an assert like lines 26-29.